### PR TITLE
Notify level publish

### PR DESCRIPTION
**Pull request by petervandivier**
Bump to version 1.5 for breaking changes. 
* Add support to Publish from config files prior to NotifyLevel 
* Add patchfile handling for version bumping/breaking changes
* Annotate requirement for Tasks to be named after home DB

`Publish-TaskFromConfig` v 1.4+ still breaks when deploying to DB version 1.40 or lower

### CODE REVIEW

File: deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+if col_length(N'scheduler.Task',N'NotifyLevelEventlog') is null
+    alter table scheduler.Task add NotifyLevelEventlog int not null default 2
+go
+
+alter table scheduler.Task alter column NotifyLevelEventlog int not null;
+go
+
+if exists( select *
```
Comment (taddison) [inline deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql]: SQL2016 has been good to us
```sql
alter table scheduler.Task drop constraint if exists DF_Task_NotifyLevelEventlog 
```

**Review by taddison — CHANGES_REQUESTED** [review]
Great additions (always knew building migration scripts was something that had to happen as we bake functionality in - this is a good format).

File: deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql (hunk 1 above)
Comment (petervandivier) [reply deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql]: Awesome to know. But the patchfile as written handles for system named defaults for those installations affected by line 2 :wink:

Comment (petervandivier) [reply deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql]: Although it now occurs to me how obviously redundant line 5 is... :sweat_smile:

Comment (taddison) [reply deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql]: The constraint name can be specified when adding a column directly, we should do that and simplify the overall script.

**Review by taddison — APPROVED** [review]

**Pull request merged**